frameworks/g-cloud-9/questions/services/scalingType.yml

```diff
@@ -12,8 +12,10 @@ type: checkboxes
 options:
   - label: Scales automatically
     value: automatic
+    filter_label: automatic
   - label: Scales with user intervention
     value: user_intervention
+    filter_label: manual through user interface
```

Comment:
Good point. I hadn't changed the wording on this one from the original draft, so assumed that 'through interface' would be ok. I'll have a think and reword it.